src/schemas/json/bacalhau-job.json

```
@@ -0,0 +1,506 @@
+{
+  "$defs": {
+    "Deal": {
+      "properties": {
+        "Concurrency": {
+          "type": "integer"
+        },
+        "Confidence": {
+          "type": "integer"
+        },
+        "MinBids": {
+          "type": "integer"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "Job": {
+      "properties": {
+        "APIVersion": {
+          "type": "string"
```

```suggestion
          "type": "string",
          "pattern": "pattern"
```

Optional fix: `pattern` can be added to check api version format.